src/core/lv_obj_style_gen.c

```diff
@@ -761,6 +761,14 @@ void lv_obj_set_style_bitmap_mask_src(lv_obj_t * obj, const lv_image_dsc_t * val
     };
     lv_obj_set_local_style_prop(obj, LV_STYLE_BITMAP_MASK_SRC, v, selector);
 }
+
+void lv_obj_set_style_rotary_sensitivity(lv_obj_t * obj, uint32_t value, lv_style_selector_t selector)
```

Comment:
> But since there is no other way to store behavioural properties for objects, "hacking" into style properties is OK, too.

It might make sense to say the sensitivity is a design choice too. :slightly_smiling_face: 
 
> Btw it would be good to have a way to store object behavioural attributes, as there could be other use cases also to store such properties. One example of behavioural attribute is "checkable", which is at the moment a state, and only some objects make use of it.

Nitpicking: it's a flag.

I was also thinking about it. Maybe storing widget properties as we store style properties? Probably we could save some RAM this way, as most of the properties are not used.

Reply:
Yeah, checkable is flag, checked is state :)

Reply:
Well, the actual flags are stored quite compact (1 bit/flag => sum 4 bytes), so at the moment I can't suggest any better option.